eclass/linux-mod-r1.eclass

```diff
@@ -138,6 +138,7 @@ if [[ ${MODULES_INITRAMFS_IUSE} ]]; then
 			sys-kernel/installkernel
 		)
 	"
+	inherit mount-boot-utils
```

Comment:
Just style, but I'd personally move it at the top of the conditional block (before `IUSE+=`) given we typically do inherits before other things.

Reply:
Makes sense, fixed, thanks!